oshi-core/src/main/java/oshi/jna/platform/mac/CoreFoundation.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021-2025 The OSHI Project Contributors
+ * SPDX-License-Identifier: MIT
+ */
+package oshi.jna.platform.mac;
+
+import com.sun.jna.Native;
+
+/**
+ * CoreFoundation class. This class should be considered non-API as it may be removed if/when its code is incorporated into the
+ * JNA project.
+ */
+public interface CoreFoundation extends com.sun.jna.platform.mac.CoreFoundation {
+
+    CoreFoundation INSTANCE = Native.load("CoreFoundation", CoreFoundation.class);
+
+    int kCFDateFormatterNoStyle = 0;
+    int kCFDateFormatterShortStyle = 1;
+    int kCFDateFormatterMediumStyle = 2;
+    int kCFDateFormatterLongStyle = 3;
+    int kCFDateFormatterFullStyle = 4;
```

Comment:
Let's make this an enum like https://github.com/java-native-access/jna/blob/ec70f9484b3c7c0adf7444b9397df71df35c6957/contrib/platform/src/com/sun/jna/platform/mac/CoreFoundation.java#L260-L274

Reply:
On second thought, looking at [the method signature](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle) we just need the constants (which should be `long` or `NativeLong`) and the shortened names:
[case fullStyle](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle/fullstyle)
[case longStyle](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle/longstyle)
[case mediumStyle](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle/mediumstyle)
[case noStyle](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle/nostyle)
[case shortStyle](https://developer.apple.com/documentation/corefoundation/cfdateformatterstyle/shortstyle)

Reply:
See implementation here:
 -  https://github.com/java-native-access/jna/pull/1672